### PR TITLE
Correct blockName in tests

### PR DESCRIPTION
**Pull request by gela-d**
/cc @dfilatov

### CODE REVIEW

File: common.blocks/checkbox-group/checkbox-group.spec.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
     'spec',
     ['checkbox-group', 'i-bem__dom', 'jquery', 'dom', 'BEMHTML', 'sinon'],
-    function(provide, RadioGroup, BEMDOM, $, dom, BEMHTML, sinon) {
+    function(provide, CheckboxGroup, BEMDOM, $, dom, BEMHTML, sinon) {
 
 describe('checkbox-group', function() {
     var checkboxGroup;
 
-    function buildRadio(bemjson) {
+    function buildCheckbox(bemjson) {
```
Comment (dfilatov) [inline common.blocks/checkbox-group/checkbox-group.spec.js]: Might be `buildCheckboxGroup`?

**Comment by gela-d** [comment]
> Might be `buildCheckboxGroup`?

@dfilatov 
Yes, sure, my mistake.
Fixed - https://github.com/bem/bem-components/commit/93f2c3022d218aaa37344258dd278c46151ace4e

**Comment by gela-d** [comment]
@dfilatov Dima, can I merge?

**Comment by dfilatov** [comment]
yes, you can )

**Pull request merged**